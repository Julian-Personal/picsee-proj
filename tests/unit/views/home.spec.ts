import { mount } from '@vue/test-utils';
import Home from '@/views/Home.vue';
import { createDefaultOptions } from '../../helper/createDefaultOptions';

describe('Home.vue', () => {
    it('should render Home well', () => {
        const wrapper = mount(Home, createDefaultOptions());
        expect(wrapper.element).toMatchSnapshot();
    });
});
