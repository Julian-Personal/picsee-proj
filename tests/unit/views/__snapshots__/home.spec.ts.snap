// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home.vue should render Home well 1`] = `
<div
  class="home"
>
  <img
    alt="Vue logo"
    src="../assets/logo.png"
  />
   
  <div
    class="row"
  >
    <div
      class="col-sm-8 col"
      style="margin: 0px auto;"
    >
      <div
        class="d-flex flex-row justify-center align-center"
      >
        <form
          class="v-form"
          novalidate="novalidate"
          style="width: 80%;"
        >
          <div
            class="v-input pr-5 v-input--hide-details theme--light v-text-field"
          >
            <div
              class="v-input__control"
            >
              <div
                class="v-input__slot"
              >
                <div
                  class="v-text-field__slot"
                >
                  <label
                    class="v-label theme--light"
                    for="input-4"
                    style="left: 0px; position: absolute;"
                  >
                    Main input
                  </label>
                  <input
                    id="input-4"
                    type="text"
                  />
                </div>
              </div>
            </div>
          </div>
        </form>
         
        <button
          class="v-btn v-btn--contained v-btn--disabled theme--light v-size--default"
          disabled="disabled"
          type="button"
        >
          <span
            class="v-btn__content"
          >
            Click 
          </span>
        </button>
      </div>
    </div>
     
    <div
      class="col-sm-8 col"
      style="margin: 0px auto;"
    >
      <!---->
    </div>
  </div>
</div>
`;
